src/test/groovy/com/avast/gradle/dockercompose/CaptureOutputTest.groovy

```diff
@@ -34,7 +36,7 @@ class CaptureOutputTest extends Specification {
         f.project.tasks.composeUp.up()
         then:
         noExceptionThrown()
-        stdout.toString().contains("web_1  | heres some output\nweb_1  | and some more")
+        stdout.toString().contains("web_1") || stdout.toString().contains("web-1")
```

Comment:
Could we please keep also checking of `| heres some output\nweb_1 | and some more` message there? 🙏 

Also, feel free to correct the typo (`heres`) 😄

Reply:
Good point.  That's fixed now and also accounts for the slight differences between v1 & v2.